Composer/packages/client/src/components/Page.tsx

```diff
@@ -64,7 +64,7 @@ export const headerContent = css`
 
 export const main = (hasRenderHeaderContent) => css`
   margin-left: 2px;
-  height: ${hasRenderHeaderContent ? 'calc(100vh - 181px)' : 'calc(100vh - 165px)'};
+  max-height: ${hasRenderHeaderContent ? 'calc(100vh - 181px)' : 'calc(100vh - 165px)'};
```

Comment:
I have issues with 181px, where is it coming from? 😄 , why not 180px?

Reply:
Yeah. I dont actually know. It's a "magic" number. The changes I made actually dont affect those. I can add a TODO to look into this.

Reply:
[Look into magic numbers in styles · Issue #6873 · microsoft/BotFramework-Composer (github.com)](https://github.com/microsoft/BotFramework-Composer/issues/6873)